linux/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 cmake_minimum_required(VERSION 3.10)
 project(runner LANGUAGES CXX)
 
+set(OPENSSL_USE_STATIC_LIBS FALSE)
```

Comment:
You sure this is correct? Regarding to https://pub.dev/packages/sqlcipher_flutter_libs#different-behavior-on-different-platforms it should be `OFF` and not `FALSE`

Reply:
They are the same. but follow the suggestion maybe better, thanks